files/galaxy/welcome.html

```diff
@@ -6,13 +6,13 @@
 <meta name="generator" content="Docutils 0.3.9: http://docutils.sourceforge.net/" />
 <link rel="stylesheet" href="style/base.css" type="text/css" />
 <style type="text/css">
-body{background-color:#EEEEFF;background-image:url(/static/images/cloud.gif);background-repeat:repeat-x;background-position: center 80px;margin:0px;}
+body{background-color:#EEEEFF;background-image:url(static/images/cloud.gif);background-repeat:repeat-x;background-position: center 80px;margin:0px;}
```

Comment:
This will not work with the standard (i.e., this role's) setup because the request will be nested under a global `static` folder, resulting in '/static/static/images/cloud.gif` URI. I'll merge it in though and just fix it in a subsequent commit but wanted to give you a heads up in case it requires a change downstream.